frontend/src/pages/Login/Login.js

```diff
@@ -50,7 +50,16 @@ const Login = (props) => {
 
   const [state, setState] = React.useState({
     email: "",
-    password: ""
+    password: "",
+    snack: {
```

Comment:
What is the snack for? Is it just for tracking errors and handling?

Reply:
Yes, the snack is for displaying any errors that was caught when attempting to login